lib/scss_lint/linter/disable_linter_reason.rb

```diff
@@ -0,0 +1,39 @@
+module SCSSLint
+  # Checks for "reason" comments above linter-disabling comments.
+  class Linter::DisableLinterReason < Linter
+    include LinterRegistry
+
+    def visit_comment(node)
+      return unless node.value.first.match(COMMAND_REGEX)
+
+      # No lint if the first line of the comment is not the command.
+      return unless comment_lines(node).first.match(COMMAND_REGEX)
```

Comment:
Is the check above necessary with this present, or are they testing something different?